src/csharp/Microsoft.Spark/ML/Feature/IDF.cs

```diff
@@ -138,17 +104,7 @@ public static IDF Load(string path)
             return WrapAsIDF(
                 SparkEnvironment.JvmBridge.CallStaticJavaMethod(s_IDFClassName, "load", path));
```

Comment:
Looks like all these ML classes share this method in common.  Is this something that can be refactored as well?

Reply:
I wasn’t sure how because it is static and returns the specific type so implementing it on the base class would need the name of the java class to call but because the implementing class doesn’t call the constructor it can’t pass the name in.

Maybe a static constructor on the implementing class, I’ll give it a go.

Reply:
I've moved load onto the base class using the static constructor, let me know if this is acceptable.

Reply:
I was mostly curious if we should.  Let's revert the most recent change, we can leave as it for now.  Everything else lgtm.

Reply:
done :)